### PR TITLE
[12.0][MIG] product_fao_fishing

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT20174

### CODE REVIEW

File: product_fao_fishing/models/product_fao_fishing_technique.py [hunk 1]
```diff
@@ -12,7 +12,6 @@ class ProductFaoFishingTechnique(models.Model):
     _parent_store = True
     _parent_order = 'name'
     _rec_name = 'complete_name'
-    _order = 'parent_left'
```
Comment (pedrobaeza) [inline product_fao_fishing/models/product_fao_fishing_technique.py]: _order = 'parent_path'

File: product_fao_fishing/models/product_fao_fishing_technique.py [hunk 2]
```diff
@@ -12,7 +12,6 @@ class ProductFaoFishingTechnique(models.Model):
     _parent_store = True
     _parent_order = 'name'
```
Comment (pedrobaeza) [inline product_fao_fishing/models/product_fao_fishing_technique.py]: You can remove this line

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
You must add a migration script that recomputes the parent_path. See the point in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0 about that

**Comment by ernestotejeda** [comment]
Changes done

**Review by pedrobaeza — APPROVED** [review]

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**